.changeset/few-oranges-complain.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/catalog-client': patch
```

Comment:
small nit pick but can we split this up into two changeset as the current text does not make much sense in the catalog-client changelog :) 

Does not have to be more than `Updated API docs`